plugins/core/editor/routes/locales.js

```diff
@@ -51,25 +60,35 @@ module.exports = {
       },
       handler: (request, h) => {
         // compute a translation.json file for use by i18next for the given language
-        let translations = {};
+        const translations = {}
+        let previewSizes
+
+        const criteria = request.auth ? request.auth.credentials : {};
+
+        // check for the get function to be backwards compatible if the config object gets
+        // passed directly instead of providing a Confidence getter
+        if (options.editorConfig.get instanceof Function) {
```

Comment:
When do we plan to remove that check again? Most times there's no need to make something backwards compatible.

Reply:
I haven't planned the removal. We could as well do it bc breaking now.